plugins/node/opentelemetry-plugin-mysql/package.json

```diff
@@ -46,7 +46,7 @@
     "@opentelemetry/test-utils": "^0.9.0",
     "@opentelemetry/tracing": "0.10.1",
```

Comment:
tracing being on 0.10.2 is what causes the `randomSpanId` issue. api/core will update to 0.10.2 on fresh installs due to being set to ^0.10.2.

looks like in related places you bumped everything to 0.10.2/^0.10.2, so i'd prob do that here as well.

Reply:
Ah right, thanks!